bHY6yh-d65839/cod-bxxqqq_bHY6yh-d65839.xml

```diff
@@ -0,0 +1,157 @@
+               <resp>TEI Encoder</resp>
+            </respStmt>
+        <respStmt>
+          <name ref="grantschreiber">Grant Schreiber</name>
+          <resp>Transcription Editor</resp>
+          <resp>TEI Encoder</resp>
+        </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 7</title>
+               <date when="2024-02-09">February 9, 2024</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+              <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2024-02-09" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="112-r"/>
+            <cb ed="#B" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d65839"><!-- p3l6-r7 -->
+            <head xml:id="bHY6yh-d65839-Hd1e108">Rubrica 7</head>
+            <head xml:id="bHY6yh-d65839-Hd1e111" type="question-title">De locutione illatiua passionis animi et electis exemplis huiusmodi</head>
+            <p xml:id="bHY6yh-d65839-d1e114">
+              De locutio<lb ed="#B" break="no"/>ne 
+              illatiua passionis animi 
+              <lb ed="#B"/>et electis exemplis huius Rubrica 7
+            </p>
+            <p xml:id="bHY6yh-d65839-d1e123">
+              <ref xml:id="bHY6yh-d65839-Rd1e120">ARistotiles rethoricorum 3 et finale capitulo
+              <lb ed="#B"/>8</ref> <quote xml:id="bHY6yh-d65839-Qd1e129">decentiam habebit <sic>loqutio</sic> 
+              <lb ed="#B"/>si sit passionis illatiua sed ali<lb ed="#B" break="no"/>quid 
+              ¶ Illatiua passionis locutio
+              <lb ed="#B"/>est si quid contumelia irascentis 
+              <lb ed="#B"/>l<sic>loqutio</sic> si autem impia et turpis 
```

Comment:
Not clear what's going on here.
Should this "l" outside the "sic" be there? or is this just a typo. If typo, I'll make the suggested change of removing it. But please double check.

```suggestion
              <lb ed="#B"/><sic>loqutio</sic> si autem impia et turpis 
```